packages/roosterjs-editor-api/lib/table/insertTable.ts

```diff
@@ -39,12 +47,46 @@ export default function insertTable(
         vtable.applyFormat(format);
         vtable.writeBack();
         editor.insertNode(fragment);

```

Comment:
I think the problem is why we can insert node into a readonly node.

Try fix insertNode() to make sure not insert under the readonly node.

Reply:
If we are inside of a readonly node what should we do? I think inserting after the readonly should be okay right?